docs/kotlin.core/overload-resolution.md

```diff
@@ -272,7 +272,7 @@ The rest of this section will try to clarify this mechanism in more detail.
 
 When an overload resolution set $S$ is selected and it contains more than one callable, the next step is to choose the most appropriate candidate from these callables.
 
-The selection process uses the [type constraint][Type constraints] system of Kotlin, in a way similar to the process of [determining function applicability][Determining function applicability for a specific call].
+The selection process uses the [type constraint][Kotlin Type constraints] system of Kotlin, in a way similar to the process of [determining function applicability][Determining function applicability for a specific call].
```

Comment:
To better follow the capitalization rules, this should be `Kotlin type constraints`